docs/ui/schema-conv/indi-table.md

```diff
@@ -0,0 +1,59 @@
+---
+layout: default
+title: Individual Table migration
+parent: Schema Conversion Workspace
+grand_parent: SMT UI
+nav_order: 5
+---
+
+# Individual Table migrations
+{: .no_toc }
+
+Spanner migration tool provides the ability to migrate a subset of tables from the source database to Spanner. This can help in a number of ways such as:
+
+1. Running a POC on a sample set of tables.
+2. Breaking up a monolithic database schema into multiple logical databases on Spanner.
+3. Avoid migration of redundant/deprecated tables from source to Spanner.
+
+Spanner migration tool supports skipping/restoring both, each table individually, and several tables in bulk.
+In the section below, we will look at how this can be configured using the Spanner migration tool UI.
+
+{: .highlight }
+Note: For Postgres minimal downtime migrations, Spanner migration tool currently does not support configuring Datastream to stream only the selected tables. All tables will be streamed, but only the selected tables will be copied into Spanner.
```

Comment:
Let's create an issue to track this and link it here

Reply:
An internal issue already exists for this. Do you mean to create a GitHub issue and link it?